packages/nitro-protocol/gas-benchmarks/gas.ts

```diff
@@ -28,17 +28,15 @@ type Path =
 export const gasRequiredTo: GasRequiredTo = {
   deployInfrastructureContracts: {
     vanillaNitro: {
-      NitroAdjudicator: 2416171, // Singleton
-      ETHAssetHolder: 1636603, // Singleton (could be more in principle)
-      ERC20AssetHolder: 1659990, // Per Token (could be more in principle)
+      NitroAdjudicator: 3954624, // Singleton
```

Comment:
Curious, what is the source of these numbers?

Reply:
All the numbers in this file are checked in the neighbouring `benchmark.test.ts` file:

https://github.com/statechannels/statechannels/blob/a755ed16e2b96550feec13ea94040e20f6d08bb7/packages/nitro-protocol/gas-benchmarks/benchmark.test.ts#L33-L37